distribution/lib/Standard/Base/0.0.0-dev/src/Data/Numbers.enso

```diff
@@ -788,6 +872,78 @@ type Integer
     floor : Integer
     floor self = @Builtin_Method "Integer.floor"
 
+    ## Truncate an `Integer` to an `Integer`, i.e. returns its argument. For
```

Comment:
We should have an `ALIAS truncate` here.

Reply:
Isn't `truncate` actually a better name overall for this method? (We could keep `int` as an `ALIAS` surely)